pkg/ansible/watches/watches.go

```diff
@@ -74,21 +76,52 @@ var (
 	manageStatusDefault                = true
 	watchDependentResourcesDefault     = true
 	watchClusterScopedResourcesDefault = false
+	selectorDefault                    = metav1.LabelSelector{}
 
 	// these are overridden by cmdline flags
 	maxWorkersDefault       = 1
 	ansibleVerbosityDefault = 2
 )
 
+// Creates, populates and returns Label Selector object during UnmarshalYAML
+func parseTemp(dls tempLabelSelector) metav1.LabelSelector {
```

Comment:
any alternative function names in mind?

Reply:
Moving forward ... this label filtering can be used for other types of operator as well?
Pls ignore this question if we already have this for helm or go.

Reply:
Yeah, would be good to make it clearer that this is for parsing the label selectors. 

wrt the second question, currently this will not be exposed to other operator types, but it could be exposed eventually